### ISSUE TITLE
Extract sequence features from BGZIPped genome sequences

### ISSUE THREAD

**Issue by lucventurini** [gpertea/gffread#36]
Currently `gffread` dies with segmentation fault when trying to extract features from a BGZIPped, indexed genome sequence.

**Comment by gpertea** [comment]
Indeed, that's not supported (yet). Shouldn't be that hard to add, I suppose - though I generally dislike adding external dependencies like bgzf or htslib unless really needed.
